.github/docker/targets/admin-tools.Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG ALPINE_IMAGE
+FROM ${ALPINE_IMAGE}
+
+ARG TARGETARCH
+
+RUN apk add --no-cache \
+    ca-certificates \
+    tzdata \
+    tini
+
+COPY --chmod=755 ./build/${TARGETARCH}/temporal /usr/local/bin/
+COPY --chmod=755 ./build/${TARGETARCH}/tdbg /usr/local/bin/
+COPY --chmod=755 ./build/${TARGETARCH}/temporal-cassandra-tool /usr/local/bin/
+COPY --chmod=755 ./build/${TARGETARCH}/temporal-sql-tool /usr/local/bin/
+COPY --chmod=755 ./build/${TARGETARCH}/temporal-elasticsearch-tool /usr/local/bin/
+
+COPY ./build/temporal/schema /etc/temporal/schema
+
+RUN addgroup -g 1000 temporal && \
+    adduser -u 1000 -G temporal -D temporal
+
+USER temporal
+WORKDIR /etc/temporal
+
+ENTRYPOINT ["tini", "--", "sleep", "infinity"]
```

Comment:
Do we need to be in `/etc/temporal` still?

Do we need `tini` here to call `sleep infinity`? Does it need to be the entrypoint? Can't we use a `CMD` for that?

Reply:
How about this?


```
CMD ["sh" "-c", "trap exit INT HUP TERM; sleep infinity"]
```